### ISSUE TITLE
Responsive Footer Design

### ISSUE THREAD

**Issue by abhijeetnishal** [WebXDAO/WebXDAO.github.io#301]
### Detailed description

Currently, the website lacks responsiveness, which hinders accessibility and usability for users accessing it from mobile devices or different screen resolutions.

Assign me this issue under GSSOC2023

### Context

This improvement will enhance the user experience, attract more visitors, and ultimately contribute to the project's growth and success.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by MAYANKpandey14** [comment]
Hey @abhijeetnishal really appreciate your spirit of contribution and collaboration but kindly read #289 as this feature is being discussed there.
Welcome to the community.

**Comment by vinzvinci** [comment]
I am closing this because it already exists. Always check current issues. Don't worry there's a lot more to contribute!
